common/utils/classnames.ts

```diff
@@ -47,12 +47,25 @@ export function font(
   return `font-${family} font-size-${size} ${overrideClasses || ''}`;
 }
 
+// TODO remove all the checks for . when we get rid of font-[color] classes
```

Comment:
nit: let's link to the ticket here https://github.com/wellcomecollection/wellcomecollection.org/issues/8418